### PR TITLE
Fix/tokens page case sensitive search

### LINKED ISSUE #2168: Tokens page: token search is case sensitive
**To reproduce:** 
1. Open Tokens page /#/account/tokens?chain=mainnet
2. Search for DAI token, but change a register for some letters in the address (like 0X6B175474E89094C44Da98b954EedeAC495271d0F)

**AR**: no tokens found

**ER**: token should be found

**Note**: this feature has been added to a token list search in #2165

### PR DESCRIPTION
**Pull request by tukantje**
# Summary

Fixes #2168

Allows for case insensitive searches in tokens page.
Adds e2e tests for this.

  # To Test

1. Open the application
2. Go to tokens page (/#/account/tokens)
3. On Goerli: search for 0XdC31eE1784292379fBB2964b3B9C4124D8F89C60
4. This should show you DAI even though there are case issues

### CODE REVIEW

File: cypress-custom/integration/token.test.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe('Tokens', () => {
```
Comment (shoom3301) [inline cypress-custom/integration/token.test.ts:1]: Awesome!

**Review by shoom3301 — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
Nice, thanks for this fix!

But actually, for a moment I though you were fixing this issue:

image
When searching for a token that's in the list by address, it shows the `import` as if it wasn't loaded.

Anyway, different issue, great job adding cypress tests!

**Comment by elena-zh** [comment]
Hey @tukantje , great job!
I think that the issue reported by @alfetopito above is related to the #2181

**Review by elena-zh — APPROVED** [review]

**Comment by tukantje** [comment]
Thank you @elena-zh @alfetopito - will address this issue in #2181 ; been working on it for a bit already 🙏

**Review by anxolin — COMMENTED** [review]
Thanks for adding tests 🙏

**Pull request merged**